service-tests/pypi.js

```diff
@@ -4,6 +4,7 @@ const Joi = require('joi');
 const ServiceTester = require('./runner/service-tester');
 const { isSemver } = require('./helpers/validators');
 const isCommaSeperatedPythonVersions = Joi.string().regex(/^([0-9]+.[0-9]+[,]?[ ]?)+$/);
+const isCommaSeperatedDjangoVersions = Joi.string().regex(/^([0-9]+.[0-9]+[,]?[ ]?)+$/);
```

Comment:
These regexes are the same. 🤔

Reply:
Yes - they're the same regex being used to validate 2 different things which is why I've assigned that regex to 2 different names. I did think about this myself and thought about:

```js
const isCommaSeperatedPythonVersions = Joi.string().regex(/^([0-9]+.[0-9]+[,]?[ ]?)+$/);
const isCommaSeperatedDjangoVersions = isCommaSeperatedPythonVersions;
```

or

```js
const isCommaSeperatedPythonOrDjangoVersions = Joi.string().regex(/^([0-9]+.[0-9]+[,]?[ ]?)+$/);
```

but I think the way I've written it make it most clear. There's no meaningful performance consideration here so it is all about readability. Do you prefer a different notation?

Reply:
Sure, makes sense. Maybe just throw in a comment with that rationale?
```js
// These regexes are the same, but defined separately for clarity.
```